codebundles/twitter-query-tweets/sli.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Metadata          Author    Shea Stewart
+Documentation     Queries Twitter to count amount of tweets within a specified time range for a specific user handle.
+Force Tags        Twitter    Social  tweet
+Suite Setup       Suite Initialization
+Library           RW.Core
+Library           RW.SocialScrape.SocialScrape
```

Comment:
Just a nitpick; Since you setup the __init__ file, you should be able to shorten this to `RW.SocialScrape`
and then use the method name as: `RW.SocialScrape.Twitter Scrape Handle`

Reply:
👍  Thanks for the review and helpful details!